var/spack/repos/builtin/packages/py-torch/package.py

```diff
@@ -260,24 +260,6 @@ class PyTorch(PythonPackage, CudaPackage):
     patch('https://github.com/pytorch/pytorch/commit/c74c0c571880df886474be297c556562e95c00e0.patch?full_index=1',
           sha256='8ff7d285e52e4718bad1ca01ceb3bb6471d7828329036bb94222717fcaa237da', when='@:1.9.1 ^cuda@11.4.100:')
 
-    @property
-    def libs(self):
-        # TODO: why doesn't `python_platlib` work here?
```

Comment:
Do you know what this comment means? It implies to me that the code in this function doesn't work (and also that this PR - which generalizes this logic - would not address the problem).

Reply:
This comment means that the `python_platlib` module attribute defined in `python`'s `setup_dependent_package` isn't found, I had to use `self.spec['python'].package.platlib` (the fully qualified name) to access it.